doc/CMakeLists.txt

```diff
@@ -32,6 +33,14 @@ IF(DEAL_II_COMPONENT_DOCUMENTATION)
     PATTERN "CMakeLists.txt" EXCLUDE
     PATTERN "doxygen" EXCLUDE
     PATTERN "*.in" EXCLUDE
+    PATTERN "news/changes.h" EXCLUDE
+    PATTERN "news/changes/*" EXCLUDE
+    )
+
+  INSTALL(FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/news/changes.h
+    DESTINATION ${DEAL_II_DOCHTML_RELDIR}/news
+    COMPONENT documentation
```

Comment:
This is not a priori wrong, but I wonder whether there's any purpose in it. These files are not really meant to be read by humans, but to be processed by doxygen. What are your thoughts?